pom.xml

```diff
@@ -1077,12 +1086,39 @@
                 <version>4.5.1</version>
             </dependency>
 
+            <dependency>
+                <groupId>io.confluent</groupId>
+                <artifactId>kafka-avro-serializer</artifactId>
+                <version>${dep.confluent.version}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>org.apache.kafka</groupId>
+                        <artifactId>kafka-clients</artifactId>
+                    </exclusion>
+                </exclusions>
+            </dependency>
+
             <dependency>
                 <groupId>io.jsonwebtoken</groupId>
                 <artifactId>jjwt</artifactId>
                 <version>0.9.0</version>
             </dependency>
 
+
+            <!-- io.confluent:kafka-avro-serializer uses multiple versions of this transitive dependency -->
```

Comment:
Let's move this comment to `presto-kafka/pom.xml`. Same for belows.

Reply:
The dependencies are transitive, they are not included directly anywhere.